hbw/app/javascript/packs/hbw/components/shared/hoc/conditions.js

```diff
@@ -61,11 +69,16 @@ export default WrappedComponent => class WithConditions extends Component {
 
   some = results => results.some(el => el === true);
 
+  fireFieldValueUpdate = (name, value) => {
+    this.props.trigger('hbw:update-value', { name, value });
```

Comment:
Does it work properly with several fields subscribed on the same variable?

Reply:
![form](https://user-images.githubusercontent.com/42524654/96452616-f1824800-1221-11eb-8b02-dfa68b266007.gif)
"1" - one field disabled, "2" - two fields disabled, "3" - group disabled